framework/test/gtest/BundleManifestTest.cpp

```diff
@@ -191,3 +217,216 @@ TEST(BundleManifestTest, ParseManifest)
   framework.Stop();
   framework.WaitForStop(std::chrono::milliseconds::zero());
 }
+
+namespace cppmicroservices {
+
+struct TestBundleAService
+{
+  virtual ~TestBundleAService() {}
+};
+
+}
+
+TEST(BundleManifestTest, DirectManifestInstall)
```

Comment:
Added test that does a manifest injection of a bundle with a statically linked second bundle in the same dll file.

Reply:
This test works perfectly as long as the CMAKE build has BUILD_SHARED_LIBS is on. If I understand this switch correctly, this turns static linking on (that is, all the bundles are statically linked to the main program, and when the main program starts, they're all automagically installed). In this case, the attempt to inject the manifest fails because the bundles have already been installed. The test that I wrote checks the case in which you have more than one bundle statically linked together in one dll. In that case, the manifest injection works properly.

So, my question is, is supporting manifest injection on a statically linked executable a requirement that we didn't call out? Or is it a non-requirement?

Reply:
do you mean calling `InstallBundles(location, manifest)` passing in a location to a EXE? If so, take a look at the API documentation for `InstallBundles(location)`. Does it support installing bundles within an EXE?